README.md

```diff
@@ -515,15 +515,15 @@ propertyValidators: {
     type: 'integer',
     minimumValue: 1,
     maximumValue: 100,
-    customValidation: function(doc, oldDoc, currentItemElement, validationItemStack) {
+    customValidation: function(doc, oldDoc, currentItemEntry, validationItemStack) {
```

Comment:
I changed the name of the `customValidation` function's third parameter throughout the pull request for consistency.